Trappist/src/Promact.Trappist.Test/TestConduct/TestConductRepositoryTest.cs

```diff
@@ -303,6 +304,55 @@ public TestConductRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
 
             Assert.True(testStatus == TestStatus.CompletedTest);            
         }
+
+        [Fact]
+        public async Task TransformationTest()
+        {
+            var testAttendee = InitializeTestAttendeeParameters();
+            await _testConductRepository.RegisterTestAttendeesAsync(testAttendee, _stringConstants.MagicString);
+            var attendeeId = await _trappistDbContext.TestAttendees.OrderBy(x => x.Email).Where(x => x.Email.Equals(testAttendee.Email)).Select(x => x.Id).FirstOrDefaultAsync();
+
+            // Creating test
+            var test = await CreateTestAsync();
+            //Creating test category
+            var categoryList = new List<DomainModel.Models.Category.Category>();
+            var category1 = CreateCategory("Mathematics");
+            await _categoryRepository.AddCategoryAsync(category1);
+            categoryList.Add(category1);
+            var categoryListAc = Mapper.Map<List<DomainModel.Models.Category.Category>, List<CategoryAC>>(categoryList);
+            categoryListAc[0].IsSelect = true;
+            await _testRepository.AddTestCategoriesAsync(test.Id, categoryListAc);
+            //Creating test questions
+            var questionList = new List<QuestionAC>
+            {
+                CreateQuestionAC(true, "Category1 type question 1", category1.Id, 1),
+                CreateQuestionAC(true, "Category1 type question 2", category1.Id, 1),
+            };
+            await _testRepository.AddTestQuestionsAsync(questionList, test.Id);
+            var testInstruction = await _testConductRepository.GetTestInstructionsAsync(_stringConstants.MagicString);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused 'testInstruction' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)